### PR TITLE
Minor edit to mention ssh keys

### PR DESCRIPTION
**Pull request by sijin**
Minor edit to mention ssh keys as alternative.

### CODE REVIEW

**Comment by TomCorwine** [comment]
I was going to submit the same pull request. The current text gives the impression of no authentication at all.

**Comment by falconxunit** [comment]
Good explication

File: security-checklist.md [hunk 1]
```diff
@@ -64,7 +64,7 @@
 - [ ] Use a decent provisioning script to create VMs in the cloud.
 - [ ] Check for machines with unwanted publicly `open ports`.
 - [ ] Check for no/default passwords for `databases` especially MongoDB & Redis.
-- [ ] Use SSH to access your machines; do not setup a password.
+- [ ] Use SSH to access your machines; do not setup a password, use SSH key-based authentication instead.
```
Comment (falconxunit) [inline security-checklist.md]: , use SSH key-based authentication instead.

File: security-checklist.md [hunk 2]
```diff
@@ -64,7 +64,7 @@
 - [ ] Use a decent provisioning script to create VMs in the cloud.
 - [ ] Check for machines with unwanted publicly `open ports`.
 - [ ] Check for no/default passwords for `databases` especially MongoDB & Redis.
```
Comment (falconxunit) [inline security-checklist.md]: databases

**Comment by abhishek-anand** [comment]
Agree, should be explicit. Merging the PR. Thanks.

**Comment by falconxunit** [comment]
What's wrong
Martin Leclerc

**Pull request merged**